extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -734,6 +734,17 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 				if (!wasCancelled) {
 					this.error(`'${command}' request failed with error.`, err);
 					const properties = this.parseErrorText(err && err.message, command);
+					/* __GDPR__
+						"languageServiceErrorResponse" : {
+							"command" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" },
+							"message" : { "classification": "CallstackOrException", "purpose": "PerformanceAndHealth" },
+							"stack" : { "classification": "CallstackOrException", "purpose": "PerformanceAndHealth" },
```

Comment:
I believe that the stack will include paths if a user is using the workspace version of typescript

Reply:
I understand that the classification "CallstackOrException" covers that.